### PR TITLE
@orta => CocoaPods update

### PR DESCRIPTION
**Pull request by ashfurrow**
Updates to the latest CocoaPods build (current master branch). 

- [x] Run in simulator
- [x] Run in device
- [x] Pass unit tests

### CODE REVIEW

**Comment by ashfurrow** [comment]
Tests are failing on iOS 7 with an iPhone 4S. Not sure why – has something to do with fonts (thought it might be frameworks, but no, this is with static libraries). 

screen shot 2015-03-09 at 5 15 36 pm

**Comment by orta** [comment]
That's super odd. Looks like the sans-serif is always coming as a serif font. You done the derived data dance?

**Comment by ashfurrow** [comment]
I have, but that was before I switched from iOS 8 -> 7, so I will do so again.

**Comment by ashfurrow** [comment]
It did no work. 

Noooooope

**Comment by orta** [comment]
does it do so in app?

**Comment by ashfurrow** [comment]
Yeah. Donna debug it now. 

screen shot 2015-03-09 at 5 27 05 pm

**Comment by ashfurrow** [comment]
Looks like the "ViewSubHeader" font is getting cached with the incorrect font beforehand.

screen shot 2015-03-09 at 5 28 49 pm

**Comment by ashfurrow** [comment]
Hmm no scratch that.

**Comment by ashfurrow** [comment]
The `itemWithKey:` method is returning "Avant@16" for the key:

```
(lldb) po _themeDictionary
{
    AltViewHeader = "Garamond@21";
    AltViewSubHeader = "Garamond@18";
    BodyText = "Garamond@16";
    ButtonFont = "Avant@14";
    DefaultNavigationAnimationAlphaTarget = "0.2";
    DefaultNavigationAnimationDuration = "0.3";
    DefaultNavigationAnimationXMovement = 8;
    DefaultNavigationAnimationYMovement = 16;
    Error = "#ca0814";
    FeedHeaderSubtitle = "Garamond@15";
    FeedHeaderTitle = "Garamond@21";
    FeedItemSeperatorBackgroundColor = "#E5E5E5";
    FeedItemSeperatorBottomMargin = 0;
    FeedItemSeperatorHeight = 1;
    FeedItemSeperatorHorizontalMargin = "-40";
    FeedItemSeperatorXOffset = 0;
    FeedItemTopMargin = 20;
    FeedSectionTitle = "Avant@13";
    FeedSectionTitleVerticalMargin = 15;
    FeedSectionTitleVerticalOffset = 0;
    LinkItemSubtitle = "Avant@10";
    LinkItemTitle = "Garamond@18";
    LoginError = "Garamond@17";
    LoginForm = "Garamond@17";
    LoginMessage = "Avant@16";
    ViewHeader = "Avant@18";
    ViewHeaderSideMargin = 120;
    ViewHeaderTopMargin = 20;
    ViewSubHeader = "Avant@16";
}
```

Looking into why now.

**Comment by ashfurrow** [comment]
Similar errors are occurring on master ([example](https://eigen-ci.s3.amazonaws.com/snapshots/53708939/index.html)).

**Comment by ashfurrow** [comment]
Looks like it, and likely some other changes, were made intentionally: https://github.com/artsy/eigen/commit/0c28d985246d83904f01e3bb822dd4774529cab3#diff-35d419fd452ddd7e8faa91cb981e7335 We probably didn't update the reference images because, well, we were expect reds anyway :\

**Comment by orta** [comment]
OK, is it only snapshots failing/

**Comment by ashfurrow** [comment]
Yup. I'll fix.

**Comment by ashfurrow** [comment]
Hmm. One snapshot test, and one other test still consistently failing.

File: Artsy Tests/ArtsyAPI+ArtworksTests.m [hunk 1]
```diff
@@ -7,37 +7,43 @@
...
+            }]
         ];
 
         [ArtsyAPI getFairsForArtwork:artwork success:^(NSArray *fairs) {
             results = fairs.copy;
         } failure:nil];
         [Expecta setAsynchronousTestTimeout:2];
-        expect(results.count).will.equal(1);
-        expect([(Fair *)results[0] fairID]).will.equal(@"organizer-and-profile");
+        expect(results.count).will.equal(2);
```
Comment (ashfurrow) [inline Artsy Tests/ArtsyAPI+ArtworksTests.m]: This needed changing with [this commit](https://github.com/artsy/eigen/commit/bbcf7e5ed6b2a2b49bb8d5e9c831cafa882c9604#diff-1053a0a09d9e3f2472f36e814ae851c5R104) but fell through the cracks due to Travis redding.

**Comment by ashfurrow** [comment]
Lawlz, so fixing the artwork API test also fixed the failing snapshot test (probably something to do with timing). This is greening locally.

**Comment by ashfurrow** [comment]
This is good for review.

File: Artsy Tests/ArtsyAPI+ArtworksTests.m (hunk 1 above)
Comment (orta) [inline Artsy Tests/ArtsyAPI+ArtworksTests.m]: :+1:

**Comment by orta** [comment]
Everything looks :+1: to me

**Pull request merged**